app/javascript/src/containers/AboutUs.jsx

```diff
@@ -0,0 +1,289 @@
+                  <Card.Content>
+                  <Card.Header>Aman Ibrahim</Card.Header>
+                  <Card.Meta>Class of 2020</Card.Meta>
+                </Card.Content>
+                <Card.Content extra>
+                <a>
+                  <Icon name='user' />
+                  Founder
+                </a>
+                </Card.Content>
+              </Card>
+
+             
+
+              <Card centered>
+                <Image src='https://i.imgur.com/9aBaBVM.jpg'/>
+                <Card.Content>
+                  <Card.Header> Vinit Parekh </Card.Header>
+                  <Card.Meta>Class of 2020</Card.Meta>
+                </Card.Content>
+                <Card.Content extra>
+                  <a>
+                    <Icon name='user' />
+                    Co-Founder
+                  </a>
+                </Card.Content>
+              </Card>
+
+              <Card centered>
+                <Image src='https://i.imgur.com/rdxlOsq.jpg' />
+                <Card.Content>
+                  <Card.Header> Rikki Kendall </Card.Header>
+                  <Card.Meta> Class of 2020</Card.Meta>
+                </Card.Content>
+                <Card.Content extra>
+                  <a>
+                    <Icon name='user' />
+                    Co-Founder
+                  </a>
+                </Card.Content>
+              </Card>
+              </Card.Group>
+
+              <Card.Group centered>
+                <Card centered>
+                  <Image rounded src='https://i.imgur.com/LGLJXsS.jpg' />
+                  <Card.Content>
+                    <Card.Header> Anesu Mafuvadze </Card.Header>
+                    <Card.Meta>Class of 2020</Card.Meta>
+                  </Card.Content>
+                  <Card.Content extra>
+                    <a>
+                      <Icon name='user' />
+                      Co-Founder
+                    </a>
+                  </Card.Content>
+                </Card>
+
+                <Card centered>
+                  <Image rounded src='https://i.imgur.com/Ks3OAV1.jpg' />
+                  <Card.Content>
+                    <Card.Header> Abdulla Shahid </Card.Header>
+                    <Card.Meta>Class of 2020</Card.Meta>
+                  </Card.Content>
+                  <Card.Content extra>
+                    <a>
+                      <Icon name='user' />
+                      Team Member
+                    </a>
+                  </Card.Content>
+                </Card>
+
+                <Card centered>
+                  <Image rounded src='https://i.imgur.com/BJNB5pt.jpg'/>
+                  <Card.Content>
+                    <Card.Header> Shamikh Hossain </Card.Header>
+                    <Card.Meta>Class of 2020</Card.Meta>
+                  </Card.Content>
+                  <Card.Content extra>
+                    <a>
+                      <Icon name='user' />
+                      Team Member
```

Comment:
Change "Team Member" and "Tern" to either "Developer", "Marketing", or "Junior Developer"